### ISSUE TITLE
Allow custom ports for connections to SQL server DBs

### ISSUE THREAD

**Issue by shdblowers** [findmypast-oss/mssql_ecto#1]
## Expected Behavior

That I can set any port for the connection in my config, like so:

```elixir
config :my_app, MyApp.Repo,
  adapter: MssqlEcto,
  hostname: "localhost"
  port: "9494"
```

## Current Behavior

Port config is ignored.

## Possible Solution

Implement code change to allow it to happen, most likely will need changes to `Mssqlex` as well.

## Context

Trying to use custom ports.

## Your Environment

* MssqlEcto version: 0.1
* Ecto version: 2.1
* Elixir version: 1.4
* Erlang version: 19
* Microsoft SQL Server version: 2016
* Operating System and version: Ubuntu Linux 16.04
* Link to your project: N/A

**Comment by jbachhardie** [comment]
You can still do ports right now by specifying `hostname: "localhost,4000"` for example, although I'm all for a PR that makes that more elegant.

**Comment by shdblowers** [comment]
Fixed by #17
